projects/noloan2/app/src/main/java/hasadna/noloan2/LawsuitPdf.java

```diff
@@ -0,0 +1,248 @@
+    private String userPhone = "טלפון משתמש";
+    private String userFax = "פקס משתמש";
+
+    // First spam company's details:
+    private String companyName = "שם החברה";
+    private String companyId = "מספר ח.פ";
+    private String companyAddress = "כתובת החברה";
+    private String companyPhone = "טלפון החברה";
+    private String companyFax = "פקס חברה";
+
+    // Second spam company's details:
+    private String company2Name = "שם החברה 2";
+    private String company2Id = "מספר ח.פ 2";
+    private String company2Address = "כתובת החברה 2";
+    private String company2Phone = "טלפון החברה 2";
+    private String company2Fax = "פקס חברה 2";
+// -------------------------------------------------------------------------------------:
+    Button btnCreate;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_lawsuit_pdf);
+        btnCreate = (Button) findViewById(R.id.button_createPDF);
+        appName = getString(R.string.app_name); // Get App's name
+
+        // Check Read/Write permissions:
+        isWriteStoragePermissionGranted();
+
+        // Read lawsuit template's content:
+        templatePath = Environment.getExternalStorageDirectory().getPath() + "/" + appName + "/" + templatePath;
+        templateContent = readTemplate();
+
+        // Init dates:
+        receivedSpamDate = new Date();
+        currentDate = new Date();
+
+        // Generate lawsuit PDF OnClick:
+        btnCreate.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                try {
+                    createPdf();
+                } catch (IOException e) {
+                    Log.w(TAG, "Error creating PDF Lawsuit");
+                    e.printStackTrace();
+                }
+            }
+        });
+    }
+
+    /**
+     * Creates a PDF of the lawusuit with all spam company & user's details.
+     * @throws IOException  Thrown when there's an error reading lawusuit template file.
+     * @throws DocumentException
+     */
+    public void createPdf() throws IOException {
+
+        // Activate license:
+        // TODO: Find a better solution for reading the licensekey file (This was a just a "quick fix" to get it working).
+        InputStream licenseIs = getResources().openRawResource(R.raw.itextkey);
+        LicenseKey.loadLicenseFile(licenseIs);
+
+        // Update date values:
+        dateAndTimeFormatter = new SimpleDateFormat(DATE_TIME_FORMAT);
+        dateAndTimeFormatter.setTimeZone(TimeZone.getTimeZone(TIME_ZONE));
+        dateFormatter = new SimpleDateFormat(DATE_FORMAT);
+        dateFormatter.setTimeZone(TimeZone.getTimeZone(TIME_ZONE));
+
+        // Create lawsuit's file name with current date:
+        String directory_path = Environment.getExternalStorageDirectory().getPath() + lawsuitOutput;
+        Date dt = new Date();
```

Comment:
You can just use `new Date()` inline:
`String targetPdf = directory_path + dateAndTimeFormatter.format(new Date()) + ".pdf";`

Reply:
Done.